ca_on_wilmot/people.py

```diff
@@ -1,27 +1,36 @@
 from utils import CanadianPerson as Person
 from utils import CanadianScraper
 
-COUNCIL_PAGE = "https://www.wilmot.ca/Modules/contact/search.aspx?s=EFHOVXSi8AOIMKMStZMNvAeQuAleQuAl"
+COUNCIL_PAGE = "https://www.wilmot.ca/en/township-office/council.aspx"
 
 
 class WilmotPersonScraper(CanadianScraper):
     def scrape(self):
         page = self.lxmlize(COUNCIL_PAGE)
 
-        councillors = page.xpath('//table[@class="contactList"]//tr')
+        councillors = page.xpath('.//table[@class="icrtAccordion"]//tr')
+        councillors = parse_counsillors(councillors)
         assert len(councillors), "No councillors found"
         for councillor in councillors:
-            name, role_district = councillor.xpath(".//button/text()")[0].split(" - ", 1)
-            if "Mayor" in role_district:
-                yield scrape_mayor(councillor, name)
-                continue
-            role, district = role_district.split(" - ")
-
+            role_name, contact_info = councillor
+            role_name = role_name.text_content().strip()
+            if "—\xa0" in role_name:
+                role, name = role_name.split("—\xa0")
+            else:
+                role, name = role_name.split("— ")
+
```

Comment:
Is this code actually necessary or did you just copy-paste it from the other scraper?